app/code/Magento/CatalogImportExport/Model/Import/Product/LinkProcessor.php

```diff
@@ -0,0 +1,340 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\CatalogImportExport\Model\Import\Product;
+
+use Magento\Catalog\Model\ResourceModel\Product\LinkFactory;
+use Magento\CatalogImportExport\Model\Import\Product;
+use Magento\Framework\App\ResourceConnection;
+use Magento\Framework\DB\Adapter\AdapterInterface;
+use Magento\ImportExport\Model\Import;
+use Magento\ImportExport\Model\ResourceModel\Helper;
+use Magento\ImportExport\Model\ResourceModel\Import\Data;
+use Psr\Log\LoggerInterface;
+
+class LinkProcessor
+{
+    /**
+     * @var \Magento\CatalogImportExport\Model\Import\Product
+     */
+    protected $_entityModel;
```

Comment:
As this is a new file could we tighten up these to private? As well as updating the naming without the _

Reply:
okay